backend/src/main/scala/cromwell/backend/standard/StandardAsyncExecutionActor.scala

```diff
@@ -293,6 +293,32 @@ trait StandardAsyncExecutionActor extends AsyncBackendJobExecutionActor with Sta
   def recoverAsync(jobId: StandardAsyncJob): Future[ExecutionHandle] = Future.fromTry(Try(recover(jobId)))
 
   /**
+    * A correct implementation should reconnect to the specified job id, and upon reconnection try to abort the job.
+    * With the understanding that the job might already be complete or aborted.
```

Comment:
I'm not the grammarian that @mcovarr is but this sentence seems like it's not a sentence.